### ISSUE TITLE
Cannot navigate to next diff in embedded diff editor

### ISSUE THREAD

**Issue by hediet** [microsoft/vscode#188934]
Testing #188537

![Code_-_Insiders_nldJ3tQoQY](https://github.com/microsoft/vscode/assets/2931520/6db4907e-b721-4720-9920-492e0ee88228)

**Comment by meganrogge** [comment]
I no longer see `Go to next difference` in the command palette. I also tried assigning a keybinding to `editor.action.diffReview.next` and that did not work. 

https://github.com/microsoft/vscode/assets/29464607/3b1c6434-8259-4f92-820f-fcfd3ff9f88e
